### ISSUE TITLE
Visualize WLAN channel occupation

### ISSUE THREAD

**Issue by shumpohl** [BornToBeRoot/NETworkManager#365]
A plot that shows how crowded certain frequency ranges are in one glance.

This is a possible realization from the vodafone router "WLAN Radar" (in german but the idea is clear):
<img width="1094" alt="WLAN_channel_visualization" src="https://user-images.githubusercontent.com/3215896/99149005-a413c200-268b-11eb-998f-72cfe64e6470.png">

**Comment by BornToBeRoot** [comment]
This? Is already available. (It's in the tab "Channels")


![image](https://user-images.githubusercontent.com/16019165/99158339-f5df3b00-26d1-11eb-822c-f48e9755df1e.png)

**Comment by shumpohl** [comment]
Ah, sorry! Exactly what I was looking for. Apparently I was too tired when looking for it. I did not notice the tabs.

**Comment by shumpohl** [comment]
@BornToBeRoot I've found multiple smaller and bigger issues with the plotting:

 - The y-axis is wrong. It 0dBm is more than -100dBm. This leads to weaker signals being plotted more prominently (besides being unphysical)
 - Channels 64 and 100 overlap in the plot although there is a gap in the frequency band.
 - The bandwidth is ignored. Some (older) 5GHz access points only use 20MHz.
 - There are issues when multiple APs habe the same SSID. I did not investigate it yet further.

Should I open a seperate issue or edit this one?

BTW: I love this software :)
